content/en/docs/pipelines/installation/overview.md

```diff
@@ -59,11 +70,14 @@ Upgrade Support (**Beta**)
 in place.
 
 Notes on specific features
-: After deployment, your Kubernetes cluster contains Kubeflow Pipelines only. 
+:
+  * After deployment, your Kubernetes cluster contains Kubeflow Pipelines only. 
```

Comment:
> "  * After deployment, your Kubernetes cluster contains Kubeflow Pipelines only. "

Suggestion:

> "  * After deployment, your Kubernetes cluster should contain Kubeflow Pipelines only. "

(As in, "you should see that the cluster contains KFP only but it's not a certainty, in case the didn't go as planned because of a typo in a command or a step was missed.")

Reply:
Consider dropping this bullet point. I think that it is sufficiently clear that standalone KFP does not include the other Kubeflow features.